playpen/all_27.sh

```diff
@@ -0,0 +1,17 @@
+for repo in pulp pulp_puppet pulp_rpm
+do
+    pushd /home/vagrant/devel/$repo
+    git checkout 2.7-dev
+    find ~/devel -name "*.pyc" -delete
+    sudo ./pulp-dev.py -I
```

Comment:
It's a fine suggestion, but that would be separate of this script since pulp-dev has been around a long time.